python/ray/air/tests/test_integration_mlflow.py

```diff
@@ -51,6 +49,11 @@ def setUp(self):
         client.create_experiment(name="existing_experiment")
         assert client.get_experiment_by_name("existing_experiment").experiment_id == "0"
 
+    def tearDown(self) -> None:
+        # Remove tune session if initialized to clean up for next test
+        tune_session._session = None
```

Comment:
We should modify the `shutdown` method in `tune.trainable.session` to remove both globals and use that instead